docs/npm.md

```diff
@@ -0,0 +1,395 @@
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "is-positive": ""
+  },
+  "devDependencies": {
+    "cachito-npm-without-deps": ""
+  },
+  "workspaces": [
+    "foo"
+  ]
+}
+```
+</details>
+
+<details>
+<summary>package-lock.json</summary>
+
+```jsonc
+cat package-lock.json
+{
+  "name": "cachito-npm-with-multiple-dep-versions",
+  "version": "1.0.0",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "cachito-npm-with-multiple-dep-versions",
+      "version": "1.0.0",
+      "license": "ISC",
+      "workspaces": [
+        "foo"
+      ],
+      "dependencies": {
+        "is-positive": ""
+      },
+      "devDependencies": {
+        "cachito-npm-without-deps": ""
+      }
+    },
+    "foo": {
+      "version": "1.0.0",
+      "license": "ISC",
+      "dependencies": {
+        "is-positive": ""
+      },
+      "devDependencies": {}
+    },
+    "foo/node_modules/is-positive": {
+      "version": "2.0.0",
+      "resolved": "file:/tmp/deps/npm/kevva/is-positive/is-positive-external-gitcommit-75dd3a181375162eda014984f2decc663199b09e.tgz",
+      "integrity": "",
+      "license": "MIT",
+      "engines": {
+        "node": ">=0.10.0"
+      }
+    },
+    "node_modules/cachito-npm-without-deps": {
+      "version": "1.0.0",
+      "resolved": "file:/tmp/deps/npm/cachito-testing/cachito-npm-without-deps/cachito-npm-without-deps-external-gitcommit-2f0ce1d7b1f8b35572d919428b965285a69583f6.tgz",
+      "dev": true
+    },
+    "node_modules/foo": {
+      "resolved": "foo",
+      "link": true
+    },
+    "node_modules/is-positive": {
+      "version": "1.0.0",
+      "resolved": "file:/tmp/deps/npm/kevva/is-positive/is-positive-external-gitcommit-1187a61f2e18cf7c11c23d61a1bd52b9fa6a5fdf.tgz",
+      "integrity": "",
+      "license": "MIT",
+      "engines": {
+        "node": ">=0.10.0"
+      }
+    }
+  }
+}
+```
+</details>
+
+<details>
+<summary>foo/package.json (workspace)</summary>
+
+```jsonc
+cat foo/package.json
+{
+  "name": "foo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+      "is-positive": ""
+  }
+}
+```
+</details>
+
+
+
+## Using fetched dependencies
+
+See also [usage.md](usage.md) for a complete example of Cachi2 usage.
+
+Cachi2 downloads the npm dependencies into the `deps/npm/` subpath of the output directory. The directory is a flat
+list of the downloaded distributions of your runtime and build dependencies.
+
+```text
+cachi2-output/deps/npm
+├── array-flatten-1.1.1.tgz
+├── bitbucket.org
+│        └── foo-testing
+│             └── cachi2-foo
+│                       └── cachi2-foo-external-gitcommit-9e164b97043a2d91bbeb992f6cc68a3d1015086a.tgz
```

Comment:
It's not always a flat list

And runtime vs. build is not really a concept for npm (unless you're using typescript, in which case some of your devDependencies are build dependencies)

Reply:
ou copy paste leftover sry